content/guides/virtualizing/kvm.md

```diff
@@ -0,0 +1,93 @@
+# Emulating Haiku in KVM
+
+VM is an ideal way of testing operating systems without having them physically installed. Installing Haiku into VM is a solution for those who don't want to set it up into a physical computer.
+
+There are several methods offered to install Haiku. These include Vmdk images, Raw images, Iso images and Anyboot images which are a hybrid of the latter two images. The Raw image (and thus the Anyboot image) is a preinstalled environment, in which the Virtual Hard Disk size cannot be customized, but going through the installation process is not required.
+
+In this guide the Haiku operating system is being run under virtual circumstances using Ubuntu 12.04 LTS, KVM, and Virt-Manager. but you can use any distribution of Linux that supports KVM.
+
+In this guide, we have used ISO and Anyboot/Raw images. The required files can be found on the Get-Haiku page of this website which is located [here](https://www.haiku-os.org/get-haiku). Both the ISO and anyboot images are available there, do select the closest mirror to enjoy higher transfer rates. Verify using the checksums to make sure that the downloaded files are not corrupted as they are big files.
+
+### Downloading KVM on Ubuntu 10.4 and above
+
+If your Ubuntu version is over 10.04, you can enter the following into the terminal and set up KVM and Virt-manager.
+
+```
+sudo apt-get install qemu-kvm libvirt-bin ubuntu-vm-builder bridge-utils
+```
+```
+sudo apt-get install virt-manager
+```
+
+### Installing and running Haiku from an ISO image
+
+The following guide will describe the installation of Haiku on Virtual Machine, using an ISO image.
+Once KVM is set up and the ISO image is downloaded, you can start the installation.
+
+First, create a new virtual machine.
+
+![image1](https://www.haiku-os.org/files/image1_0.png)
```

Comment:
Since this document will be available as https://www.haiku-os.org/guides/virtualizing/kvm I _think_ «**`[image1](https://www.haiku-os.org/files/image1_0.png)`**» should be replaced by «**`[image1](../../static/images/image1_0.png)`**» (or something similar) to have a "relative" image.
Well, I guess you will need to choose another name for the image: maybe:
* kvm-image1.png in /static/images/ (in which case you would need to use «**`[image1](../../static/images/kvm-image1.png)`**»);
* _or_ image1.png in /static/images/kvm/ (in which case you would need to use «**`[image1](../../static/images/kvm/image1.png)`**»).

I guess this will need to be discussed.
Of course, similar changes would be required for all the other images.